crates/oxc_linter/src/config/config_builder.rs

```diff
@@ -309,36 +287,35 @@ fn get_name(plugin_name: &str, rule_name: &str) -> CompactStr {
     }
 }
 
-impl TryFrom<Oxlintrc> for LinterBuilder {
-    type Error = LinterBuilderError;
+impl TryFrom<Oxlintrc> for ConfigStoreBuilder {
+    type Error = ConfigBuilderError;
 
     #[inline]
     fn try_from(oxlintrc: Oxlintrc) -> Result<Self, Self::Error> {
         Ok(Self::from_oxlintrc(false, oxlintrc))
     }
 }
 
-impl fmt::Debug for LinterBuilder {
+impl fmt::Debug for ConfigStoreBuilder {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("LinterBuilder")
+        f.debug_struct("ConfigBuilder")
```

Comment:
We should probably be consistent here:

```suggestion
        f.debug_struct("ConfigStoreBuilder")
```

Reply:
find and replace lol